data/courses.yaml

```
@@ -56,6 +74,45 @@
   summary: >
     The goal of the course is to teach the basics of Lean to first year undergraduate students in a double major program in mathematics and computer science.
   year: 2023
+- name: Discrete Mathematics for Computer Science and Engineering
+  instructor: Kevin Sullivan
+  contact: sullivan@virginia.edu
+  institution: University of Virginia, Dept. of Computer Science
+  website: https://computingfoundations.org
+  environment: Coursework is supported by a VS-Code+Docker remote container.
+  lean_version: 4
+  tags: ['mathematics','computer science', 'programming-first', 'discrete']
```

```suggestion
  tags: ['mathematics','computer science', 'intro to proof']
```
Perhaps "functional programming" too? Trying to reuse existing tags rather than introducing new ones.